### DISCUSSION TITLE
Master Unlocker 1.39.0

### DISCUSSION THREAD

**Discussion by jhowkNx** [jhowkNx/Master-Unlocker#53] [Announcements]
### Issues?

**Comment by cloudcreatordotio** [comment]
I can't find the source code, only a compiled lua script. There were some issues with the Agency Contract unlock in the 1.38 that I would like to fix

**Reply by jhowkNx** [reply]
> I can't find the source code, only a compiled lua script. There were some issues with the Agency Contract unlock in the 1.38 that I would like to fix

Unfortunately, we had to opt for that security...but can you communicate what problem you are having, if possible in detail.

**Reply by cloudcreatordotio** [reply]
the problem is with the Agency stats.  It sets everything to 500 and total contracts to 510.  The math doesn't add up.  There are five different types of missions, so that should be 2500+ total missions. Maybe you should set the total number to 2510 instead of 510. Or maybe set each to 100 and the leave the number for the total contracts completed to 510 (100 for each type and 10 Dre contracts).

**Reply by jhowkNx** [reply]
There are actually 6 stats that were set to 500, and the one that controls the total amount of quests was also 500, the setting for it has now been set to 3000 to make it balanced.

**Reply by cloudcreatordotio** [reply]
Is this change in the current version 1.39.0? If not, when will it be released?

**Reply by jhowkNx** [reply]
This change is in version 1.39.1
